routes/new_members.rb

```diff
@@ -1,6 +1,16 @@
 require 'models/event'
 require 'models/event_signup'
 require 'classes/emailer'
+require 'recaptcha'
+
+Recaptcha.configure do |config|
```

Comment:
Check how to hide these, we should probably even change these because its very common to web scrape for keys